### PR TITLE
PR: adds section headings for linking to terms #42

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] Adding section headings and semantic line breaks #42

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab please take a look when you have 10 mins spare to read through. Not urgent. Thanks.

**Review by SimonLab — COMMENTED** [review]
I'm not sure why these two sections (CSS box model, and CRUD) aren't display properly:
image

https://github.com/dwyl/technical-glossary/blob/c3ae752c7f1484055dc4a55859595f64c910586f/README.md

@nelsonic do you have an ideas what is the issue here?

**Comment by nelsonic** [comment]
Markdown fail. Will fix when I’m back at my desk. 👌🏻

**Comment by nelsonic** [comment]
@SimonLab https://github.com/dwyl/technical-glossary/blob/11bd1006d6d21e7463c54ad4c3d1a3c78cd87694/README.md#css-box-model

image

Thanks. 👍

File: README.md [hunk 1]
```diff
@@ -3,100 +3,180 @@
...
+E.g. when you login to your email you use a password to authenticate your
+identity. 
+Other methods for authentication include using an ID card or
+fingerprint/biometric method.
 
-### How?
+> **Note**: the authentication experience 
+> can be a significant source 
+> of frustration both as person _using_ 
+> a digitial product and for the _engineers_ building it.
```
Comment (SimonLab) [inline README.md]:
```suggestion
> a digital product and for the _engineers_ building it.
```

File: README.md [hunk 2]
```diff
@@ -171,7 +279,19 @@ x = "hello";
 console.log(x)     // logs 'hello'
 ```
 
-**[Higher-order Function](https://en.wikipedia.org/wiki/Higher-order_function)** - Functions that take other functions as parameters and/or return a function. Some examples of these types of functions are the [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) and [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) method used for arrays. They both accept a function to use on the elements in the given array.
+This is obviously confusing/undesireable code,
```
Comment (SimonLab) [inline README.md]:
```suggestion
This is obviously confusing/undesirable code,
```

File: README.md [hunk 3]
```diff
@@ -3,100 +3,180 @@
...
-different domains using Amazon S3. For more about origin policies see
+See:
+https://en.wikipedia.org/wiki/CSS_box_model
+### Customer relationship management (CRM)
+
+A CRM system allows organisations to
+manage customer relationships 
+and the data and information associated with them.
+Typically a CRM is used for storing contact data 
+and the log of communication between a business and its' cusotmers.
```
Comment (SimonLab) [inline README.md]:
```suggestion
and the log of communication between a business and its' customers.
```

File: README.md [hunk 4]
```diff
@@ -187,29 +307,65 @@ function func (value) {
...
 console.log(twice(func, 7)); // 13
 ```
-Not to be confused with pure functions, [this comment](https://github.com/dwyl/learn-redux/issues/40#issuecomment-272489779) explains the difference between the variations.
 
-**Hypertext Transfer Protocol (HTTP)** - the set of rules for transferring files
-(text, graphic images, sound, video, and other multimedia files) on the World
-Wide Web. When you enter http:// in your address bar in front of the domain, it
-tells the browser to connect over HTTP.
+Not to be confused with "pure" functions.
+To undersand the difference,
```
Comment (SimonLab) [inline README.md]:
```suggestion
To understand the difference,
```

**Review by SimonLab — COMMENTED** [review]
Thanks for fixing the markdown @nelsonic, just spotted a few typos

**Comment by nelsonic** [comment]
@SimonLab thanks for fixing the typos. 👍 
_Really_ need a long-term solution for this. https://github.com/dwyl/dev-setup/issues/45 ...

Hope you have a great day. ☀️

**Review by SimonLab — APPROVED** [review]
Links up to date, thanks!

**Pull request merged**